### PR TITLE
Feature/geolocation

### PR DESCRIPTION
**Pull request by wlt30**

### CODE REVIEW

File: app/src/main/java/ca/klapstein/baudit/data/Record.java [hunk 1]
```diff
@@ -25,26 +25,30 @@
     private GeoLocation geoLocation;
     private ArrayList<BodyPhotoCoords> bodyPhotoCoords = new ArrayList<>();
     private ArrayList<String> keywords = new ArrayList<>();
-    private UUID recordId;
+    private UUID recordID;
```
Comment (nklapste) [inline app/src/main/java/ca/klapstein/baudit/data/Record.java]: Is `problemId` renamed aswell for consistency?

File: app/src/main/java/ca/klapstein/baudit/activities/MapAllProblemsActivity.java [hunk 2]
```diff
@@ -87,12 +109,73 @@ public void onLowMemory() {
...
+        for (Problem problem:problemTreeSet){
+            RecordTreeSet recordTreeSet = problem.getRecordTreeSet();
+            for(Record record: recordTreeSet){
+                LatLng marker = new LatLng(record.getGeoLocation().getLat(),record.getGeoLocation().getLon());
+                googleMap.addMarker(new MarkerOptions().position(marker).title(record.getTitle()));
+            }
+        }
+    }
+
+    public void testPopulateMap(GoogleMap googleMap){
```
Comment (nklapste) [inline app/src/main/java/ca/klapstein/baudit/activities/MapAllProblemsActivity.java]: We should move tests into their own instrumented test files. If you want me to spearhead that let me know.

File: app/src/main/java/ca/klapstein/baudit/activities/PatientHomeActivity.java [hunk 3]
```diff
@@ -232,5 +239,6 @@ public void setProblemDateText(String date) {
         public void setProblemDescriptionText(String description) {
             descriptionView.setText(description);
         }
+
```
Comment (nklapste) [inline app/src/main/java/ca/klapstein/baudit/activities/PatientHomeActivity.java]: Remove unneeded newline

File: app/src/main/java/ca/klapstein/baudit/activities/MapAllProblemsActivity.java [hunk 4]
```diff
@@ -36,6 +57,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         mapView = findViewById(R.id.map_all_problems_map);
         mapView.onCreate(mapViewBundle);
         mapView.getMapAsync(this);
+
```
Comment (nklapste) [inline app/src/main/java/ca/klapstein/baudit/activities/MapAllProblemsActivity.java]: Remove unneeded newline

File: app/src/main/java/ca/klapstein/baudit/data/Record.java (hunk 1 above)
Comment (thomaslorincz) [reply app/src/main/java/ca/klapstein/baudit/data/Record.java]: In camelCase, it is common practice to keep only the first letter of an acronym capitalized.
e.g. problemId, recordId, parsedHtml, etc.

**Pull request closed without merging**